students/seongsu/westagram/users/views.py

```diff
@@ -53,15 +54,16 @@ def post(self, request):
         try:
             email    = user_data['email']
             password = user_data['password']
-            user = User.objects.get(email = email)
+            user     = User.objects.get(email = email)
             
-            if user.password != password:
+            if bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')) == False:
```

Comment:
파이썬 PEP 8 프로그래밍 권고에는 비교문 관련해서 어떤 형식을 사용하라고 안내되어 있습니다.

- True/False를 비교할 때는 if문에 변수만 사용하도록 한다.
- 동등성(단순히 값만 비교하는 경우)을 검증할 때는 == 혹은 != 같은 연산자를 사용하도록 한다.
- 동일성(객체의 고유 메모리 주소로 비교하는 경우)을 검증할 때는 is나 is not을 사용한다.

즉 아래의 경우는 True/False만을 비교하기 때문에 == False는 붙일 필요가 없습니다.
```suggestion
            if bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
```

Reply:
false 대신 not 붙여서 수정 했습니다